assets/js/keys.js

```diff
@@ -1,7 +1,9 @@
-const config = {
-  openWeatherApiKey: "9E92A9EV239V29V82V2",
-  xRapidAPIKey: "2870710a70mshb9dfd7b0cc1c38ap18047cjsn644f8c604a16",
-  xRapidAPIHost: "wft-geo-db.p.rapidapi.com",
-};
-
-var apiKey = "jykXqNRW0Y62n8S6SXW4C5ctkG3caX7l";
\ No newline at end of file
+var apiKey = "jykXqNRW0Y62n8S6SXW4C5ctkG3caX7l";
```

Comment:
What is this key for?

Reply:
for the currency API

Reply:
Ok, so please rename it so that it reflects this.

Reply:
ok